### PR TITLE
Feature cart number

### PR DESCRIPTION
**Pull request by cl21484952**
Changelog
- Example add cart number `ตะกร้าสินค้า (1)`

_Figure 1 - Cart number_
image

### CODE REVIEW

**Comment by codenuk** [comment]
IMG_C78C20C6016D-1
I Think. This beautiful

File: src/components/navbar/index.js [hunk 1]
```diff
@@ -49,7 +60,7 @@ const NavBarComponent = ({ history }) => {
                 </button>
                 <button>
                     <ShoppingCart />
-                    ตะกร้าสินค้า
+                    ตะกร้าสินค้า {itemCountInCart ? ("(" + itemCountInCart + ")") : (<></>)}
```
Comment (codenuk) [inline src/components/navbar/index.js]: Not beautiful

**Review by codenuk — CHANGES_REQUESTED** [review]

**Comment by codenuk** [comment]
Just expand width

**Comment by cl21484952** [comment]
_Figure 3 - Width 90px with 2 characters_
image

_Figure 4 - Width 90px with 3 characters_
image

**Comment by codenuk** [comment]
If order have three character add plus at end character

**Comment by cl21484952** [comment]
_Figure 5 - >99 items_
image

_Figure 6 - 1~99 items_
image

_Figure 7 - 0 items / non-integer
image

**Review by codenuk — APPROVED** [review]

**Pull request merged**